ssl_test.go

```diff
@@ -86,6 +87,10 @@ func TestSSLVerifyFull(t *testing.T) {
 	if err == nil {
 		t.Fatal("expected error")
 	}
+	// in Go 1.20 the error is wrapped in tls.CertificateVerificationError
+	if errWrapped := errors.Unwrap(err); errWrapped != nil {
```

Comment:
could this use `errors.As` so that the test also checks that the outer error is `CertificateVerificationError`?